NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -1449,6 +1479,19 @@ static VOID ParaNdis_CleanupContext(PARANDIS_ADAPTER *pContext)
 
     virtio_device_shutdown(&pContext->IODevice);
 
+#ifdef MAX_FRAGMENTS_IN_ONE_NB
+    pContext->SendCopyPages.ForEach([](CNdisSharedMemory* e) {
+        e->~CNdisSharedMemory();
+        NdisFreeMemory(e, 0, 0);
+    }
+    );
+    pContext->SendCopyPagesInUsed.ForEach([](CNdisSharedMemory* e) {
+        e->~CNdisSharedMemory();
+        NdisFreeMemory(e, 0, 0);
+    }
+    );
+#endif
+
```

Comment:
style: no new line in between endif and }

Reply:
ok, i will delete this new line